spec/system/account_spec.rb

```diff
@@ -38,11 +40,26 @@
       end
     end
 
+    describe "update locales" do
+      it "shows the list of locales" do
+        find("#user_locale").click
+        if organization.available_locales.size == 1
```

Comment:
Please add a second case. 
First test should test with multiple locales
Second test should test with one locale